### PR TITLE
Update PyNE Feedstock to make a working lite version

### PR DESCRIPTION
**Pull request by kkiesling**
This PR should be a working recipe for a PyNE "lite" conda package. "Lite" = not including the MOAB and DAGMC dependencies (those to come in a future PR). 

Recipe and build updates:
* removes MOAB and DAGMC specifications
* removes unnecessary packages for build/host/run so that only bare minimum are included
* upgraded to be compatible with conda build 3
* added additional maintainers for the pyne-feedstock who are active developers on PyNE

This can replace PR #36 (fixes issue #35). I believe this also fixes version incompatibilities as described in #31. 

PR Co-authors: @bam241 @opotowsky @gonuke

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* ~~[] Reset the build number to `0` (if the version changed)~~
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bam241** [comment]
@conda-forge-admin, please rerender

**Comment by bam241** [comment]
@conda-forge-admin, please rerender

**Comment by bam241** [comment]
@conda-forge-admin, please rerender

**Comment by kkiesling** [comment]
@conda-forge-admin, please rerender

**Comment by kkiesling** [comment]
I believe this PR is ready for review @scopatz!

File: recipe/build.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ if [ "$(uname)" == "Darwin" ]; then
 else
   skiprpath=""
 fi
-export FC=gfortran
+#export FC=gfortran
```
Comment (gonuke) [inline recipe/build.sh]: Please delete this line

**Comment by kkiesling** [comment]
Also added the following people as maintainers in the recipe/meta.yaml file. If you do not want to be listed as a maintainer, please let us know and I can remove you:

* gonuke
* bam241
* kkiesling
* a-co
* opotowsky
* katyhuff
* pshriwise

**Comment by gonuke** [comment]
Did you figure out why `jinja2` is needed in this lightweight build?

**Comment by kkiesling** [comment]
The cram tests failed without jinja2

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,66 +1,46 @@
...
+{% set sha256 = "19e104a05aa56ef2e747f5e4a1c4cffefdb57a592a300823265ba72a1b55d4f3" %}
 
 package:
   name: pyne
   version: {{ version }}
 
 source:
   fn: pyne-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/{{ org }}/pyne/archive/{{ version }}.tar.gz
+  url: https://github.com/{{ org }}/pyne/archive/{{ version }}-rc1.tar.gz
```
Comment (scopatz) [inline recipe/meta.yaml]: Please see the very new instructions for releasing rcs: https://github.com/conda-forge/cfep/blob/master/cfep-05.md

File: recipe/meta.yaml [hunk 3]
```diff
@@ -75,3 +55,10 @@ about:
 extra:
   recipe-maintainers:
     - scopatz
+    - gonuke
+    - bam241
+    - kkiesling
+    - a-co
+    - opotowsky
+    - katyhuff
+    - pshriwise
```
Comment (scopatz) [inline recipe/meta.yaml]: @gonuke @bam241 @a-co  @opotowsky @katyhuff @pshriwise - are you OK with being maintainers?

**Comment by scopatz** [comment]
Amazing work here @kkiesling!

File: recipe/meta.yaml (hunk 3 above)
Comment (bam241) [reply recipe/meta.yaml]: I am OK to be a maintainer

File: recipe/meta.yaml (hunk 2 above)
Comment (gonuke) [reply recipe/meta.yaml]: Thanks @scopatz - we'll review this.  Part of our reason for issuing an RC was to allow us to dig into the conda-forge workflow without waiting to figure out a PyNE release.  We are hoping/imagining that a release may replace this RC soon.

Nevertheless, we'll look at the docs.

File: recipe/meta.yaml (hunk 3 above)
Comment (gonuke) [reply recipe/meta.yaml:64]: I'm ok with being a maintainer as well (in case the emoji isn't good enough)

**Comment by kkiesling** [comment]
Just realized we are waiting on people to consent to being a maintainer for this to get merged. We have identified the following people as a potential maintainers. Please state whether you do or do not want to be listed as a PR. Thanks!

@gonuke - confirmed
@bam241 - confirmed
@kkiesling - confirmed
@a-co
@opotowsky
@katyhuff
@pshriwise

**Comment by kkiesling** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 3 above)
Comment (opotowsky) [reply recipe/meta.yaml:64]: I'm OK with being a maintainer

**Comment by gonuke** [comment]
Maybe we should remove some of these proposed maintainers for now.  Once a few of us are added as maintainers, it will be easier to add others later on.

**Comment by gonuke** [comment]
Note that the removed maintainers were people who never agreed to maintain in the first place and were correctly identified in this original PR.  This PR has been updated to roll back from the early merge of #37

**Comment by kkiesling** [comment]
Uhh I did not request reviews on this PR and I am not sure why it appears that I did right now..

**Comment by gonuke** [comment]
Now that we are maintainers - we have permission to merge this ourselves!

**Pull request merged**